spec/models/app_config_spec.rb

```diff
@@ -44,4 +51,24 @@
     end
     @ac.favicon_content_type = 'image/vnd.microsoft.icon'
   end
+
+  context '.contact_email' do
+    it 'returns the contact e-mail if it is set' do
+      @ac.contact_link_location = 'contact@example.com'
+      @ac.save
+
+      expect(AppConfig.contact_email).to eq('contact@example.com')
+
+      AppConfig.delete_all
```

Comment:
ok! Sounds good!

On Mon, Mar 28, 2016 at 8:23 PM, Oren Kanner <notifications@github.com>
wrote:

> In spec/models/app_config_spec.rb
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_YaleSTC_reservations_pull_1529-23discussion-5Fr57656297&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=fGUlyXMXUllcq1Lzipj1Qfu_NL-lk9lC6GgNV_p2VOg&m=WzTiZRIKhQmCt0b1YIJXGGu0KL6Mgs470C3Eu2LGSaI&s=1ez9R_FJ8zVfZQYwvOGabEryJ_w6atZacSEzkKbGIqo&e=>
> :
>
> > @@ -44,4 +51,24 @@
> >      end
> >      @ac.favicon_content_type = 'image/vnd.microsoft.icon'
> >    end
> > +
> > +  context '.contact_email' do
> > +    it 'returns the contact e-mail if it is set' do
> > +      @ac.contact_link_location = 'contact@example.com'
> > +      @ac.save
> > +
> > +      expect(AppConfig.contact_email).to eq('contact@example.com')
> > +
> > +      AppConfig.delete_all
>
> before(:all) runs *once* before all of the tests in the block, if we were
> running it before(:each) that would be the same but this follows the Four
> Phase Test
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__robots.thoughtbot.com_four-2Dphase-2Dtest&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=fGUlyXMXUllcq1Lzipj1Qfu_NL-lk9lC6GgNV_p2VOg&m=WzTiZRIKhQmCt0b1YIJXGGu0KL6Mgs470C3Eu2LGSaI&s=CMLnzWJ78mpk9NTVqQLx_zBWYlv94XoGkUKWxgIGCG0&e=>
> model (with AppConfig.delete_all being "teardown").
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_YaleSTC_reservations_pull_1529_files_25b7573af7fe6135c094bdc99078426873f0122c-23r57656297&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=fGUlyXMXUllcq1Lzipj1Qfu_NL-lk9lC6GgNV_p2VOg&m=WzTiZRIKhQmCt0b1YIJXGGu0KL6Mgs470C3Eu2LGSaI&s=uhjGWqphR7x0m4Kd7QljLHZly6I01TvR9-sP8Qcjof0&e=>
>